pkg/kubelet/volumemanager/populator/desired_state_of_world_populator.go

```diff
@@ -322,7 +308,7 @@ func (dswp *desiredStateOfWorldPopulator) processPodVolumes(
 		}
 
 		// Add volume to desired state of world
-		_, err = dswp.desiredStateOfWorld.AddPodToVolume(
+		uniqueVolumeName, err := dswp.desiredStateOfWorld.AddPodToVolume(
 			uniquePodName, pod, volumeSpec, podVolume.Name, volumeGIDValue, seLinuxContainerContexts[podVolume.Name])
```

Comment:
Now that I see it again, I can see `AddPodToVolume` returns an empty string on error and that empty string gets passed to `checkVolumeFSResize` below (even on error!). I think it is wrong.

Reply:
Thanks! I think we should just skip `checkVolumeFSResize` on error. If `AddPodToVolume` failed, `getUniqueVolumeName`(removed in this PR) should return `!exists` anyway.